packages/core/src/layer.ts

```diff
@@ -18,7 +18,7 @@ export function useLayerLifecycle(
 ) {
   useEffect(
     function addLayer() {
-      const container = context.layerContainer ?? context.map
+      const container = context.layerContainer || context.map
```

Comment:
I think it would make more sense to update the babel config and ensure that this is compiled in the final build rather than try to change the code itself, right?

Reply:
i think that would be the better solution too 🙂